### DISCUSSION TITLE
Why?

### DISCUSSION THREAD

**Discussion by Terrinn** [3kh0/ext-remover#207] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
From a students point of view, why are you making this. I assume that you understand that most people keeping an eye on this thread are twelve year olds. If you understand that students can get in major trouble at school and even get law enforcement involved why are you continuously making new ways to bypass the schools blocks. Most kids do not understand toll something on their permanent record can have on their lives.

**Comment by ghost** [comment]
Because we're bored and make projects. I personally didn't do it for the students, I did it because I like messing with software. Anyways, this is a WAY BETTER alternative to.... _Illegal software changes_, so you should be OK with this

**Comment by synapt0** [comment]
It's to prove that Chromebooks are a sad excuse for a school computer.

**Reply by rwg100** [reply]
they use chromebooks because they cheap and easy to manage. it takes way more software to manage lets say a windows laptop. also chromebooks are much more durable than a windows computer bc literally plastic and rubber.

**Reply by fnfgo** [reply]
> they use chromebooks because they cheap and easy to manage. it takes way more software to manage lets say a windows laptop. also chromebooks are much more durable than a windows computer bc literally plastic and rubber.

Even public schools have enough money to afford windows computers for us and chromebooks are not durable

**Comment by brobcoolguy** [comment]
i honestly dont care, no offense

**Comment by rwg100** [comment]
its not a crime if you don't get caught !

oh and we are just manipulating bugs in chrome so no harm done.

**Comment by Aka-but-nice** [comment]
Buddy, what you mean, "its not a crime if you don't get caught,". Like if you kill someone its a crime.

**Reply by fnfgo** [reply]
> Buddy, what you mean, "its not a crime if you don't get caught,". Like if you kill someone its a crime.

not unless someone finds out

**Reply by Aka-but-nice** [reply]
So your saying that if you kill someone your good and have no reason to go to jail

**Reply by fnfgo** [reply]
No im saying that you have no reason to go to jail if no one finds out. And also for a good reason. Like how i strangled my dad for not coming back with the milk.

**Comment by piplup7575** [comment]
Hey!
As people have stated in other threads, **only do this if you know you won't get in trouble**! This exploit was created by Bypassi, who, as you can see, has created many exploits:
![image](https://user-images.githubusercontent.com/88395302/208244926-1c6b5845-6c11-4719-a0f8-c35b57dd5b14.png)
The main purpose of us, students, creating these exploits is not to get in trouble, but instead trying to get sysadmins to fix these bugs, so they can't be used for more nefarious exploits.
Every exploit you see in that screenshot has been created in an attempt to get sysadmins to patch it, because they could be used to create a living hell for any users on a network.
For example, LTBEEF, this exploit, was actually reported, **by the creator**, to Chromium, in an attempt to get it patched!
I hope this helps clear up any confusion.
Have a great day!
-Piplup

**Reply by brobcoolguy** [reply]
yo where can i get those exploits??
